features/admin.issuer-usage-scope.v1/api/use-issuer-usage-scope-configs.ts

```
@@ -0,0 +1,133 @@
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { AsgardeoSPAClient, HttpClientInstance } from "@asgardeo/auth-react";
+import { store } from "@wso2is/admin.core.v1/store";
+import { IdentityAppsApiException } from "@wso2is/core/exceptions";
+import { HttpMethods } from "@wso2is/core/models";
+import { AxiosError, AxiosRequestConfig, AxiosResponse } from "axios";
+import { IssuerUsageScopeConfigConstants } from "../constants/issuer-usage-scope-configuration";
+import { IssuerUsageScopeConfigAPIResponseInterface } from "../models/issuer-usage-scope-configuration";
+import { IdentityAppsError } from "@wso2is/core/dist/types/errors/identity-apps-error";
```

ESLint

[error] '@wso2is/core/dist/types/errors/identity-apps-error' import is restricted from being used by a pattern.

(no-restricted-imports)